src/App.jsx

```diff
@@ -14,70 +15,71 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const activeGood = 'has-background-success-light';
```

Comment:
```suggestion
  const activeGoodClassName = 'has-background-success-light';
```
to better describe what is it, because it's not a good (like `Jam`)

just FYI